assets/js/types/type-defs/cart.ts

```diff
@@ -67,7 +67,7 @@ export interface CartShippingPackageShippingRate extends CurrencyInfo {
 }
 
 export interface CartShippingRate {
-	package_id: number;
+	package_id: string | number;
```

Comment:
didn't we spend a lot of effort with Subscriptions to move away from string as package ids, did it surface back?

Reply:
Strings are valid afaik. The type hint is wrong.